templates/about.html

```diff
@@ -1023,6 +1027,19 @@ <h3 id="publish">Publishing</h3>
 </pre>
 </li>
 
+<li id="meetup-rsvp" class="question">How do I RSVP to an event on Meetup?</li>
+<li class="answer">
+<p>Just post an RSVP to the event's URL! For example:</p>
+<pre>
+&lt;div class="h-entry"&gt;
```

Comment:
also, very minor nit, you'll want to syntax highlight the mf2 and link URLs here like in the other doc items. the one immediately above this one is a good example.

Reply:
Done - good spot